manifests/upstream/canary-checker/base/kustomization.yaml

```diff
@@ -1,15 +1,16 @@
 resources:
-  - https://raw.githubusercontent.com/flanksource/canary-checker/v0.38.30/config/deploy/base.yaml
+  - https://raw.githubusercontent.com/flanksource/canary-checker/v0.38.67/config/deploy/base.yaml
 images:
   - name: docker.io/flanksource/canary-checker:latest
-    newTag: '{{.canaryChecker.version | default "v0.38.30" }}'
+    newTag: '{{.canaryChecker.version | default "v0.38.67" }}'
+namespace: platform-system
 patchesStrategicMerge:
   - |-
     apiVersion: apps/v1
     kind: Deployment
     metadata:
       name: canary-checker
-      namespace: platform-system
+      namespace: canary-checker
```

Comment:
I don't think we want to change the default deployment from platform-system to canary-checker

Reply:
We're not. It's just that the patch is applied before the namespace is changed in kustomize, so for these to apply, they need to match the original namespace